From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20Offenh=C3=A4user?= <offenhaeuser@protonmail.com>
Date: Sat, 11 Feb 2023 02:30:10 +0100
Subject: [PATCH] Be a bit more lenient with matching savegame directories

Our sscanf() implementation failed to match this case. Making it more
inclusive should be fine, since invalid savegame directories probably
won't contain the right files anyway.
---
 gemrb/core/SaveGameIterator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gemrb/core/SaveGameIterator.h b/gemrb/core/SaveGameIterator.h
index 4d1550eddcd2f656205170131a04f570570430f9..e93b171123c68e41e607518e6c4ffc93e455f92d 100644
--- a/gemrb/core/SaveGameIterator.h
+++ b/gemrb/core/SaveGameIterator.h
@@ -29,7 +29,7 @@
 
 namespace GemRB {
 
-#define SAVEGAME_DIRECTORY_MATCHER "%d - %[A-Za-z0-9- _+*#%&|()=!?':;]"
+#define SAVEGAME_DIRECTORY_MATCHER "%d - %s"
 
 class GEM_EXPORT SaveGameIterator {
 private:
